### ISSUE TITLE
[Feature] g_elimination_enableItems

### ISSUE THREAD

**Issue by NeonKnightOA** [OpenArena/gamecode#86]
Idea taken from [this post on the OA forums](http://openarena.ws/board/index.php?topic=5399.0). After getting used to the code, it shouldn't be that different, plus we have already an antecedent of a similar gamemode working, and it's [Unreal Tournament 4's (Team) Showdown mode](https://www.youtube.com/watch?v=cSxpCuBxEdc).

Implementation would be similar to Sago's suggestion: a single bitflag cvar.
**g_elimination_enableItems & 1 (ELIMINATION_ITEMS_ENABLEWEAPONS)** = enable
**g_elimination_enableItems & 2 (ELIMINATION_ITEMS_ENABLEHEALTHARMOR)** = but no health/armor
**g_elimination_enableItems & 4 (ELIMINATION_ITEMS_ENABLEPOWERUPS)** = but no power ups

**Comment by The-Gig** [comment]
But in that case, what would a value of 5 do?

If it will be a bitflag variable, then maybe each value may *add* something instead, allowing maximum customization?
E.g.
1=add weapons/ammo boxes
2=add heath/armor bonuses
4=add powerups (also runes in case of g_runes 1?)

So to have everything one would set it to 7 (1+2+4)?

(On a side note, wondering if ui3 may allow to manage bit field settings in a way more comfortable than ""cycle between all possible combinations by repeatedly clicking").

**Comment by NeonKnightOA** [comment]
Yep, my bad, I don't know what I was thinking when I wrote that.

**Comment by The-Gig** [comment]
Just a small nitpicking... IIRC all elimination* tuning options do not have the initial "g_", probably to save some chars considering that "elimination" itself is quite a long word.
So, the cvar might be something like elimination_enableItems, elimination_keepItems, or something similar, without the g_.

**Comment by Danil-Dm** [comment]
In my mod there is cvar elimination_items it includes items.
You can take ideas from my mod
https://github.com/Danil-Dm/openarena

**Comment by NeonKnightOA** [comment]
I saw Ratmod implemented this very feature. I might have to take a look.
